src/pagination.ts

```diff
@@ -114,25 +127,33 @@ export class CursorPaginator<T> {
 
   /**
    * Execute a single query for the requested page of results.
+   *
+   * @throws a runtime error when aliased `order` columns do not exist in the query.
```

Comment:
This is likely a personal preference, but I like to hoist the throws docblock comments throughout.